src/Chainweb/Storage/Table/RocksDB.hs

```diff
@@ -341,59 +313,12 @@ newTable
     -> RocksDbTable k v
 newTable db valCodec keyCodec namespace
     | any (B8.any (\x -> x `elem` ['$', '%', '/'])) namespace
-        = error $ "Data.CAS.RocksDb.newTable: invalid character in table namespace: " <> sshow namespace
+        = error $ "Chainweb.Storage.Table.RocksDb.newTable: invalid character in table namespace: " <> sshow namespace
     | otherwise
         = RocksDbTable valCodec keyCodec ns (_rocksDbHandle db)
   where
     ns = _rocksDbNamespace db <> "-" <> B.intercalate "/" namespace
```

Comment:
Any reason why this `INLINE` pragma disappeared? I see a lot of disappearing `INLINE` pragmas in this file, could be nice to add a line to the PR description to explain why.

Reply:
Done